app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -26,12 +26,21 @@ def index
         end
       end
 
+      def show
+        record = ParticipantDeclaration.find_by(id: params[:id])
```

Comment:
`ParticipantDeclaration.for_lead_provider(cpd_lead_provider).find_by(id: params[:id])`?

We don't want to show the declaration of Ambition to UCL (or anyone else), even if the chances they get the right UID is slim.

Reply:
Yeah, good point, updated in 01e5060 and also added to swagger docs